tests/curve_tests.ts

```diff
@@ -83,6 +84,36 @@ const simple = createTest(
   }
 );
 
+const openingDir = createTest(
+  ['c4/8, f5, d5, g5', { stem: 'up' }], // beamGroup1
+  ['d6/8, f5, d5, g5', { stem: 'down' }], // beamGroup2
+  (f, notes) => {
+    f.Curve({
+      from: notes[0],
+      to: notes[3],
+      options: {
+        cps: [
+          { x: 0, y: 10 },
+          { x: 0, y: 50 },
+        ],
+        openingDirection: 'up',
```

Comment:
In the test image, it looks like it curves up then down. Does it look this way without those `cps`?

It seems like in order to honor `openingDirection`, vexflow will have to control `cps`. What have you found when experimenting with it?

Reply:
Without the cps on the first curve

![image](https://github.com/user-attachments/assets/849f2af8-77a6-4a4a-87c5-8177f1b029ed)

Reply:
I have not experimented much :)